benchmark/benchmark.cr

```diff
@@ -86,8 +86,6 @@ end
 
 Mosquito::Runner.stop wait: true
 
-
-
```

Comment:
This type of change is unnecessary, and it sabotages your goal here. Forego style changes, whitespace changes, and any other restructuring of the codebase. If something needs to change, go for it -- but that change needs to be explained in the commit, or in a comment on the code.